uefi/src/fs/file_system.rs

```diff
@@ -278,19 +302,24 @@ impl<'a> FileSystem<'a> {
         mode: UefiFileMode,
         is_dir: bool,
     ) -> FileSystemResult<UefiFileHandle> {
-        let path = NormalizedPath::new("\\", path)?;
-        log::debug!("normalized path: {path}");
+        validate_path(path)?;
```

Comment:
Why is `validate_path` used here but not any of the other `FileSystem` methods?

Reply:
Because, If I'm not mistaken, every single FS-method uses the open() call under-the-hood.

Reply:
Ah, that certainly makes sense!